### PR TITLE
Add docs about Fleet enrollment tokens

### LINKED ISSUE #272: Add new topic about enrollment tokens
Add a topic to the Fleet docs that decsribes:

What an enrollment token is, how it works, how to create a token, how to revoke a token, etc

### PR DESCRIPTION
**Pull request by dedemorton**
Closes #272.

Preview link: https://observability-docs_823.docs-preview.app.elstc.co/guide/en/fleet/master/fleet-enrollment-tokens.html

Note that the tasks are written to work on 7.13. We will update the docs with the 7.14 UI changes after merging and backporting.

### CODE REVIEW

**Comment by dedemorton** [comment]
@ruflin Including you as a reviewer here because some of the info about enrollment tokens was taken from the info you wrote. I know we need to add more docs about the security model. Main thing here is having something that users can refer to when they are looking at the UI and want to understand more about enrollment tokens.

File: docs/en/ingest-management/security/enrollment-tokens.asciidoc [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+* A communication API key
++
+This API key is used to communicate with the {fleet-server}. It has only the
+permissions needed to communicate with the {fleet-server}. If the API key is
+invalid, {fleet-server} stops communicating with the {agent}.
+
+// TODO: Convert the following steps to 7.14 after this topic is merged and
+// backported to 7.13.
+
+//QUESTION: Do we want to document the API calls for creating enrollment tokens
```
Comment (ruflin) [inline docs/en/ingest-management/security/enrollment-tokens.asciidoc:29]: I would expect in most cases there user never has to manually create enrollment tokens but creates policies and with it the enrollment token. I would rather focus on getting the Kibana API docs in place.

File: docs/en/ingest-management/security/enrollment-tokens.asciidoc [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+. Find the token you want to revoke in the list and click the *Revoke token*
+icon.
++
+[role="screenshot"]
+image::images/revoke-token.png[Enrollment tokens tab with Revoke token highlighted]
+
+. Click *Revoke enrollment token*. You can no longer use this token to enroll
+{agent}s.
+
+Any {agent}s enrolled through the token will stop communicating with
```
Comment (ruflin) [inline docs/en/ingest-management/security/enrollment-tokens.asciidoc:84]: @nchaulet Is this the case? If the enrollment token is invalidated also all the ouput and communication keys related to it are invalidated? If yes, we should validate if this is the expected behaviour.

Comment (nchaulet) [reply docs/en/ingest-management/security/enrollment-tokens.asciidoc:84]: No it's not the case enrolled agent will continue to work

Comment (dedemorton) [reply docs/en/ingest-management/security/enrollment-tokens.asciidoc:84]: @nchaulet My agent definitely stopped working, but I didn't look at the logs closely to see what was going on. Might have been a weird coincidence. Should I retest, or are you sure that the agent continues to work?

Comment (dedemorton) [reply docs/en/ingest-management/security/enrollment-tokens.asciidoc:84]: I've removed the sentence. If it turns out to be correct, I can add it back in.

File: docs/en/ingest-management/security/enrollment-tokens.asciidoc [hunk 3]
```diff
@@ -0,0 +1,81 @@
+[[fleet-enrollment-tokens]]
+= {fleet} enrollment tokens
+
+An enrollment token is an {es} API key that you use to enroll one or more
```
Comment (EricDavisX) [inline docs/en/ingest-management/security/enrollment-tokens.asciidoc:4]:
```suggestion
A {fleet} enrollment token is an {es} API key that you use to enroll one or more
```

File: docs/en/ingest-management/security/enrollment-tokens.asciidoc [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+. In {fleet}, click *Agents > Enrollment tokens*.
+
+. Find the token you want to revoke in the list and click the *Revoke token*
+icon.
++
+[role="screenshot"]
+image::images/revoke-token.png[Enrollment tokens tab with Revoke token highlighted]
+
+. Click *Revoke enrollment token*. You can no longer use this token to enroll
+{agent}s.
```
Comment (EricDavisX) [inline docs/en/ingest-management/security/enrollment-tokens.asciidoc:79]:
```suggestion
{agent}s, however, the currently enrolled Agents will continue to function.
```

**Review by EricDavisX — APPROVED** [review]
I like this, thanks.  you can take the suggestions and work them or skip it.  not a deal breaker.

**Pull request merged**